cypress/integration/app.spec.js

```diff
@@ -0,0 +1,8 @@
+/// <reference types="cypress" />
+
+describe('Homepage', () => {
+  it('displays a welcome text', () => {
+    cy.visit('/');
+    cy.contains('Welcome');
```

Comment:
How about 
- configuring the `data-test-id` for the [test id attribute](https://nimblehq.co/compass/development/code-conventions/javascript/cypress/#element-selector) ([here](https://github.com/nimblehq/varun-web/blob/develop/cypress/support/configure-testing-library.ts) is some example on Varun project)
- adding a test ID attribute to the current UI
- [selecting the element with the id](https://docs.cypress.io/guides/references/best-practices#Selecting-Elements) and test that it contains the expected text

so that we're sure that it would work once we have the real UI with the test id attribute 🙌🏼

Reply:
that's a good idea, done in https://github.com/tyrro/react-survey/pull/47/commits/4e0fa630b33db2a061f0f2458d4a80d8866693e7 👍